packages/kbn-securitysolution-ecs/src/index.ts

```diff
@@ -122,6 +122,7 @@ export interface EcsSecurityExtension {
   suricata?: SuricataEcs;
   system?: SystemEcs;
   timestamp?: string;
+  tags?: string[];
```

Comment:
I was kind of expecting this to be in the same "level" as `workflow_status` like `kibana.alert.tags`. Curious if there's pros/cons to where we chose to place it?

Reply:
The `tags` field already exists on the alerts as data schema in this location, so we're just making use of it now. We had a few discussions in the advanced correlations meetings early on and decided using an existing field that was already set for a purpose such as this would be the best way forward rather than creating something new or using another nested tags field when we already have the nested rule tag field in the schema.

Reply:
Well, this comment turned out to be more insightful than expected huh